<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entrepot</class>
 <widget class="QDialog" name="Entrepot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Entrep√¥ts</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Entrepot{
    border-image: url(&quot;C:/projet_QT/projet_achats/image000/fond02.jpg&quot;) 0 0 0 0 stretch stretch;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBoxAjout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>811</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
background-image: url(&quot;C:/projet_QT/projet_achats/image000/fond03.jpg&quot;);
background-repeat: no-repeat;
background-position: center;
background-size: cover;
color: white;
front-weight: bold;
font-seize: 14px;
}</string>
   </property>
   <property name="title">
    <string>Ajouter un Entrep√¥t</string>
   </property>
   <widget class="QLabel" name="labelRefEntrepot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: qlineargradient(
    x1:0, y1:0, x2:1, y2:1,
    stop:0   #22a9d6,
    stop:0.5 #1dd3b0,
    stop:1   #22a9d6
);
border-radius: 15px;
border: none;
padding: 10px 20px;
color: white;
font-size: 15px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>R√©f√©rence :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRefEntrepot">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdresse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: qlineargradient(
    x1:0, y1:0, x2:1, y2:1,
    stop:0   #22a9d6,
    stop:0.5 #1dd3b0,
    stop:1   #22a9d6
);
border-radius: 15px;
border: none;
padding: 10px 20px;
color: white;
font-size: 15px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Adresse :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAdresse">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAjouter">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #29bf12;
border-radius: 15px;
border: none;
padding: 10px 20px;
color: white;
font-size: 10px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>üì• Enregistrer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxListe">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>811</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
background-image: url(&quot;C:/projet_QT/projet_achats/image000/fond03.jpg&quot;);
background-repeat: no-repeat;
background-position: center;
background-size: cover;
color: white;
front-weight: bold;
font-seize: 14px;
}</string>
   </property>
   <property name="title">
    <string>Liste des Entrep√¥ts</string>
   </property>
   <widget class="QTableView" name="tableViewEntrepot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>791</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView{
 background-color: White;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSupprimer">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: red;
border-radius: 15px;
border: none;
padding: 10px 20px;
color: white;
font-size: 10px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>üóëÔ∏è Supprimer la s√©lection</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {
    border-image: url(&quot;C:/projet_QT/projet_achats/image000/fond03.jpg&quot;) 0 0 0 0 stretch stretch;
    border: none;
    padding: 10px 20px;
    color: white;
    font-weight: bold;
}</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Gestion des Entrep√¥ts :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
